SESSION_FACILITATOR_GUIDE.md

```diff
@@ -1,34 +1,28 @@
-# 💁🏽 Session Facilitator Guide
+# 💁🏽 FAQ and Session Facilitator Guide
 
-This document is intended to provide a helpful starting point or reference for running sessions during summits.
-The new learnings and tools from a summit will ideally be PR'd in when each is concluded.
+This document is intended to provide a helpful starting point or reference for running sessions during Summits.
+
+The new learnings and tools from a Summit will be PR'd in when each is concluded.
 
 🎉 Lists are included 🎉
 
-## 📅 in the days leading up to the session
-- [ ] Set a chair/facilitator for the session
-- [ ] (optional) if your session is open ended where a broader issue is discussed, you may choose to prepare [tcq](https://tcq.app/) to facilitate discussion
+## Tips
 
-## 🛫 pre-flight checklist (when about to begin the session)
-- [ ] The facilitator is the person whose name is listed in the agenda. Facilitators will need to make sure the group walks out of the room with actionable items.
-- [ ] If not pre-arranged, ask the table for volunteer note taker.
-- [ ] Have a person be take charge of each tool being used for the session. For a full list of tools see below.
-- [ ] Go to Zoom, start the call.
-- [ ] In Zoom, make sure the link to the zoom session is posted on the relevant thread in [nodejs/summit](https://github.com/nodejs/summit/issues/).
-- [ ] In Zoom, start recording the session. If the available Zoom accounts are maxed out, use Hangouts.
-- 
-- [ ] (Optional) Go to [Otter](https://otter.ai), start recording the audio.
+* If you joined a Zoom call as a host, then leave it, the meeting will end for all. To avoid this, make sure another person has is the host before leaving.
+* How to share video with audio on a Zoom call: [video walkthrough](https://soapbox.wistia.com/videos/Y7CcmHs3w3).
+* If not pre-arranged, ask the table for volunteer note-taker.
+* Post the URL to the Zoom meeting on the relevant Issue in [nodejs/summit](https://github.com/nodejs/summit/issues/).
+* If agreed, record the session.
```

Comment:
I would recommend to have it as a reminder for all the speakers, or  even better set them to auto-record.